File: app/src/main/java/com/example/simplehabitassignment/network/response/LoginResponse.java
package com.example.simplehabitassignment.network.response;


import com.example.simplehabitassignment.data.vos.LoginUserVO;
import com.google.gson.annotations.SerializedName;

public class LoginResponse {

    @SerializedName("login_user")
    private LoginUserVO loginUser;

    public LoginUserVO getLoginUser() {
        return loginUser;
    }
}

File: app/src/main/java/com/example/simplehabitassignment/network/response/LoginResponse.kt
package com.example.simplehabitassignment.network.response


import com.example.simplehabitassignment.data.vos.LoginUserVO
import com.google.gson.annotations.SerializedName

class LoginResponse {

    @SerializedName("login_user")
    val loginUser: LoginUserVO? = null
}
